libs/external-router/routing/src/guard/existence.guard.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable, Inject } from '@angular/core';
+import {
+	CanActivate,
+	ActivatedRouteSnapshot,
+	Router,
+	RouterStateSnapshot,
+	UrlTree,
+} from '@angular/router';
+import { Observable, of } from 'rxjs';
+import { catchError, map, switchMap } from 'rxjs/operators';
+
+import {
+	DaffExternalRouterDriverInterface,
+	DaffExternalRouterDriver,
+} from '@daffodil/external-router/driver';
+import {
+	DaffExternalRouter,
+	DaffExternalRouterConfiguration,
+	DAFF_EXTERNAL_ROUTER_CONFIG
+} from '@daffodil/external-router';
+
+import { daffConvertToPath } from '../helper/convert-to-path';
+
+@Injectable({
+	providedIn: 'root',
+})
+export class DaffExternalRouterExistenceGuard implements CanActivate {
```

Comment:
This needs a doc

Reply:
https://github.com/graycoreio/daffodil/pull/1240/commits/e84f901398d30c4af371fb257714725339f3640a

Reply:
"for guarding the wildcard of an Angular App"

Reply:
edfbddcaaa4b4d01bff83499060fdc74e6e40d6d